etna/lib/etna/clients/magma/workflows/create_project_from_json_workflow.rb

```diff
@@ -139,20 +149,33 @@ def update_magma_attributes
           end
         end
 
-        def create!
+        def setup_janus_project!
           puts "Creating Janus project."
           create_janus_project
           puts "Done! Adding you as an administrator on the project."
           add_janus_user
           update_janus_permissions
           update_magma_client_token
-          puts "Done! Creating the project in Magma."
+          puts "Done with setting up the project in Janus!"
+        end
+
+        def setup_magma_project!
+          puts "Creating the project in Magma."
           create_magma_project
+          puts "Done! Creating all the models and attributes in Magma."
           create_magma_models
           ensure_magma_tree
+          puts "Done! Adding your new project record."
           create_magma_project_record
           update_magma_attributes
-          puts "All complete! You can visit Janus to refresh your token, then log into any app to manage your data."
+        end
+
+        def create!
+          setup_janus_project!
+          setup_magma_project!
```

Comment:
Also, note from today's standup -- sounds like we want to remove the `setup_janus_project!` behavior anyways for non-superusers, so could just remove that method.

Reply:
Yeah, so I came away with a different impression.

This command remains for _us_ to setup an empty project.  The other commands get more mature for copying models (from other projects) and editing them.

In other words, there would never be a single "bring up a project with all its pieces" command.

Reply:
I think that makes sense, but it'd be nice to keep the "make all models for project in one file" command available for the users.